lib/apiservers/portlayer/restapi/handlers/storage_handlers_test.go

```diff
@@ -17,23 +17,23 @@ package handlers
 import (
 	"context"
 	"fmt"
-	"io"
 	"net/http"
 	"net/url"
 	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
-	"github.com/vmware/govmomi/vim25/mo"
 	"github.com/vmware/vic/lib/apiservers/portlayer/models"
 	"github.com/vmware/vic/lib/apiservers/portlayer/restapi/operations/storage"
-	"github.com/vmware/vic/lib/archive"
 	"github.com/vmware/vic/lib/constants"
-	spl "github.com/vmware/vic/lib/portlayer/storage"
+	"github.com/vmware/vic/lib/portlayer/storage/image"
+	icache "github.com/vmware/vic/lib/portlayer/storage/image/cache"
+	imagemock "github.com/vmware/vic/lib/portlayer/storage/image/mock"
```

Comment:
nit: `icache` and `imagemock` is... incongruous. It results in code like `icache.NewLookupCache(imagemock.NewMockDataStore(nil))`

Any reason not to be consistent? (`icache.NewLookupCache(imock.NewMockDataStore(nil)),` or `imagecache.NewLookupCache(imagemock.NewMockDataStore(nil)),`)